adhocracy4/projects/migrations/0043_migrate_topics_to_m2m_topics.py

```diff
@@ -11,7 +11,7 @@ def add_topics_to_m2m_table(apps, schema_editor):
         project = apps.get_model("a4projects", "Project")
         topic = apps.get_model("a4projects", "Topic")
         for project in project.objects.all():
-            for topic_code in project.topics:
+            for topic_code in project.topics.split(","):
```

Comment:
@goapunk it could worth to add a `try/except` here in case there is no comma, or topic for projects